interactive_templates/templates/v2/analysis/study_definition.py

```diff
@@ -18,8 +18,8 @@
 codelist_1_type = params["codelist_1_type"]
 codelist_2_path = params["codelist_2_path"]
 codelist_2_type = params["codelist_2_type"]
-time_value = int(params["time_value"])
-time_ever = params["time_ever"]
+time_ever = params["time_ever"].lower() == "true"
+time_value = int(params["time_value"]) if not time_ever else 0
```

Comment:
Using 0 here makes me nervous, because we're also using it to represent "current month only". I don't know if we should use something obviously wrong like -1 instead, so any errors will be easier to spot? I haven't looked through the code fully though and I'm happy to follow Louis's opinion. Maybe it's just needs to be a smell that we come back to later for now.

Reply:
This is what @LFISHER7 said we should use as a default in this cases.

It basically boils down to "- 0 days" in the date maths